playbooks/roles/elasticsearch/tasks/elasticsearch_post_install.yml

```diff
@@ -0,0 +1,81 @@
+---
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Deploy ElasticSearch configuration files
+  template:
+    src: "{{ item }}"
+    dest: "/etc/elasticsearch/{{ item }}"
+    owner: "root"
+    group: "root"
+  register: results
+  with_items:
+    - elasticsearch.yml
+    - logging.yml
+  tags:
+    - elasticsearch-post-install
+
+- name: Deploy ElasticSearch service configuration file
+  template:
+    src: "{{ item }}"
+    dest: "/etc/default/{{ item }}"
+    owner: "root"
+    group: "root"
+  register: results
+  with_items:
+    - elasticsearch.j2
+  tags:
+    - elasticsearch-post-install
+
+- name: Restart ElasticSearch service
+  service: name=elasticsearch state=restarted pattern=elasticsearch
+enabled=yes
+  when: results|changed
```

Comment:
I think we should remove all the "when: results|changed" and use a handler - I've done that with the logstash role. Just remove the "registers" and all the "when" lines and add a notify once the handler is created.